docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst

```diff
@@ -0,0 +1,240 @@
+Sediment erosion and Transport from
+===================================
+
+In this notebook we implement the erosion and sediment transport
```

Comment:
only erosion - no sediment transport. It assumes infinite amount of sediment in each cell, and only calcs. vertical erosion (we didn't have time to calc. the rest).